oggm/core/preprocessing/inversion.py

```diff
@@ -171,25 +171,25 @@ def invert_parabolic_bed(gdir, glen_a=cfg.A, fs=0., write=True):
                     out_thick[i] = 0.
 
             # Check for thick to width ratio (should ne be too large)
-            ratio = out_thick / w  # there's no 0 width so we're good
-            pno = np.where(ratio > max_ratio)
-            if len(pno[0]) > 0:
-                ratio[pno] = np.NaN
-                ratio = utils.interp_nans(ratio, default=max_ratio)
-                out_thick = w * ratio
+            # ratio = out_thick / w  # there's no 0 width so we're good
+            # pno = np.where(ratio > max_ratio)
+            # if len(pno[0]) > 0:
+            #     ratio[pno] = np.NaN
+            #     ratio = utils.interp_nans(ratio, default=max_ratio)
+            #     out_thick = w * ratio
 
             # TODO: last thicknesses can be noisy sometimes: interpolate?
             if cl['is_last']:
                 out_thick[-4:-1] = np.NaN
                 out_thick = utils.interp_nans(out_thick)
 
```

Comment:
I think this should be commented, too. This interpolation is not meaningful for calving glaciers either.